From 7a777f6d6f358b8ab97baae82be03ab704d1bd1c Mon Sep 17 00:00:00 2001
From: Maxime Ripard <maxime.ripard@free-electrons.com>
Date: Thu, 15 Oct 2015 22:04:05 +0200
Subject: [PATCH 003/600] mmc: Add generic Kconfig option

Add a generic Kconfig option for the CONFIG_MMC option that was used before
in the configuration headers.

Since all the architectures need to be converted to that first, depend on
an non-existent config option that will be extended with architectures that
use that option.

Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
Reviewed-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 drivers/mmc/Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/drivers/mmc/Kconfig b/drivers/mmc/Kconfig
index 6277f92..d3d7d91 100644
--- a/drivers/mmc/Kconfig
+++ b/drivers/mmc/Kconfig
@@ -1,5 +1,11 @@
 menu "MMC Host controller Support"
 
+config MMC
+	bool "Enable MMC support"
+	depends on UNUSED
+	help
+	  TODO: Move all architectures to use this option
+
 config DM_MMC
 	bool "Enable MMC controllers using Driver Model"
 	depends on DM
-- 
2.5.0

